src/Blueprint/Blueprint/Models/PSBlueprintAssignment.cs

```diff
@@ -59,7 +59,12 @@ internal static PSBlueprintAssignment FromAssignment(Assignment assignment)
                 BlueprintId = assignment.BlueprintId,
                 ProvisioningState = PSAssignmentProvisioningState.Unknown,
                 Status = new PSAssignmentStatus(),
-                Locks = new PSAssignmentLockSettings {Mode = PSLockMode.None},
+                Locks = new PSAssignmentLockSettings
+                {
+                    Mode = PSLockMode.None,
+                    ExcludedActions = (List<string>)assignment.Locks.ExcludedActions,
```

Comment:
nit, use ToList().

Reply:
Okay

Reply:
I thought we agreed to expose ExcludedPrincipals and ExcludedActions as array of strings and no type conversion is needed then.

Reply:
From what I saw, the SDK exposes ExcludedActions and ExcludedPrincipals as IList<string>. I wasn't sure if I could use an IList directly.

Reply:
Did you run into any errors if using IList directly?

Reply:
I don't think I faced any errors. Just wasn't sure if it was a good practice.